### PR TITLE
Patch: Working Vagrant Bild on PHP7

### PR DESCRIPTION
**Pull request by bobrik717**
| Q             | A
| ------------- | ---
| Is bugfix?    | yes/no
| New feature?  | yes/no
| Breaks BC?    | yes/no
| Tests pass?   | yes/no
| Fixed issues  | comma-separated list of tickets # fixed by the PR, if any

### CODE REVIEW

**Comment by bobrik717** [comment]
Working Vagrant Bild on PHP7

**Comment by cebe** [comment]
can you explain a bit more about what you are doing here? a lot of the changes seem unrelated to php7 and also wrong.

**Comment by bobrik717** [comment]
when deploying the project was a lot of mistakes, especially composer could not Installed all the dependencies because he needed php7. I rewrote sh halyards so vagrant to unfold php7 and added the ability to work under one domain. Now if you deploy the project sufficient to establish a vagrant and virtualbox.

File: Vagrantfile [hunk 1]
```diff
@@ -2,8 +2,8 @@ require 'yaml'
 require 'fileutils'
 
 domains = {
-  frontend: 'y2aa-frontend.dev',
-  backend:  'y2aa-backend.dev'
+  frontend: 'local.site.com',
+  backend:  'local.site.com/admin'
```
Comment (samdark) [inline Vagrantfile]: Doesn't look right. Should be 2 separate hosts.

File: Vagrantfile [hunk 2]
```diff
@@ -18,7 +18,7 @@ options = YAML.load_file config[:local]
 
 # check github token
 if options['github_token'].nil? || options['github_token'].to_s.length != 40
-  puts "You must place REAL GitHub token into configuration:\n/yii2-app-advanced/vagrant/config/vagrant-local.yml"
+  puts "You must place REAL GitHub token into configuration:\n/yii2-app-advancded/vagrant/config/vagrant-local.yml"
```
Comment (samdark) [inline Vagrantfile]: Looks like typo introduced.

File: backend/config/main.php [hunk 3]
```diff
@@ -15,6 +15,7 @@
     'components' => [
         'request' => [
             'csrfParam' => '_csrf-backend',
+            'baseUrl' => '/admin'
```
Comment (samdark) [inline backend/config/main.php]: This isn't needed if hosts are separate.

File: vagrant/config/vagrant-local.example.yml [hunk 4]
```diff
@@ -10,7 +10,7 @@ timezone: Europe/London
 box_check_update: false
 
 # Virtual machine name
-machine_name: y2aa
+machine_name: site
```
Comment (samdark) [inline vagrant/config/vagrant-local.example.yml]: Why?

File: vagrant/nginx/app.conf [hunk 5]
```diff
@@ -1,77 +1,46 @@
...
-       #fastcgi_pass   127.0.0.1:9000;
-       fastcgi_pass unix:/var/run/php5-fpm.sock;
-       try_files $uri =404;
-   }
-
-   location ~ /\.(ht|svn|git) {
+   location ~* \.(htaccess|htpasswd|svn|git) {
        deny all;
    }
+   location /admin {
```
Comment (samdark) [inline vagrant/nginx/app.conf]: It should be separate host as it was before.

**Comment by rugabarbo** [comment]
PR #236 solves problem of php upgrade. So we can reject and close this PR (#227).

**Comment by samdark** [comment]
@bobrik717 sorry for rejecting but it doesn't fit current setup.

**Pull request closed without merging**